WcaOnRails/app/views/admin/_delayed_job_runner.html.erb

```diff
@@ -0,0 +1,28 @@
+<% job ||= nil %>
+<% trigger_link ||= nil %>
+
+<h2><%= "Controls for #{job.name}" %></h2>
+
+<%= yield %>
+
+<% if job.respond_to?(:reason_not_to_run) && job.reason_not_to_run %>
+  <%= alert :warning, job.reason_not_to_run %>
+<% else %>
+  <% if job.in_progress? %>
+    <%= alert :info, "The job is running. Thanks for checking =)" %>
+  <% elsif job.is_a? TimedApplicationJob %>
```

Comment:
Any reason why only CAD is TimedApplicationJob and exports (dev & results) are not TimedApplicationJob?

Reply:
Because until now we didn't need it. `Timed` in this case does not mean that the Job is timed on a schedule, but rather that we're measuring the time how long the job takes. And I'm not sure if we need that for these exports. What's your opinion?

Reply:
It was not there earlier as well, so as per our PHP migration project, definitely it's not needed.

But if it's just changing the class name or too less effort (maybe 3-4 lines of codes max), and if it doesn't add any extra cost, it will be nice to have I guess.

Currently, all WRT members knows that the CAD takes around 4-5 minutes to run. So, if at some point we see it is taking more time like 8 minutes, we will know something is wrong. Similarly, if the time of these two exports are available, eventually WRT members will know the time taken for those scripts and maybe inform WST if the time is increasing consistently, or if the time has became unusually large.